Makefile

```diff
@@ -199,4 +211,30 @@ publish-helm: ## Helm publish
 	helm push ./kro-${RELEASE_VERSION}.tgz oci://${HELM_IMAGE}
 
 .PHONY:
-release: build-image publish-image package-helm publish-helm
\ No newline at end of file
+release: build-image publish-image package-helm publish-helm
+
+##@ Deployment
+
+ifndef ignore-not-found
+  ignore-not-found = true
+endif
+
+.PHONY: install
+install: manifests ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	$(KUBECTL) apply -f ./helm/crds
+
+.PHONY: uninstall
+uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config
+	$(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f ./helm/crds
+
+.PHONY: deploy-kind
+deploy-kind:
+	$(KIND) delete clusters ${KIND_CLUSTER_NAME} || true
+	$(KIND) create cluster --name ${KIND_CLUSTER_NAME}
+	$(KUBECTL) create namespace kro-system
+	make install
+	make build-image # TODO: how do you kind.local here ?
+	kind load docker-image ${KO_DOCKER_REPO}:${RELEASE_VERSION} --name ${KIND_CLUSTER_NAME}
+	helm template kro ./helm --namespace kro-system --set image.pullPolicy=Never --set image.repository=${KO_DOCKER_REPO} --set image.tag=${RELEASE_VERSION} | $(KUBECTL) apply -f -
```

Comment:
I added the helm template ko support in the last PR to allow for 


   ```sh
    helm template kro ./helm \
      --namespace kro-system \
      --set image.pullPolicy=Never \
      --set image.ko=true | ko apply -f -
    ```

Reply:
Will try that.

Reply:
Does ko add the image using kind load image

Reply:
apparently it does.